app/views/invitations/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <!--Descriptive page name, messages and instructions-->
```

Comment:
Sure!

1. Log in as admin
2. In the "Invitations" menu, choose "Invite New Users"
3. Under "Track invitations," enter a username , e.g. testy, and press "Go"
4. Click the name "testy" to go to testy's "Create more invitations for this user" page
5. Fill in "Number of invitations" with some number and press "Create"
6. Repeat steps 2-3 to get updated search results for testy's invitations
7. Click the token for one of the invitations you just gave testy -- that will take you to an ugly, detailed view of the invitation
8. Fill in "Enter an email address" with an email address and press "Update Invitation"

The important thing is "Enter an email address" needs to have been blank to start with -- if there was already an address in there, it works fine. Also, weirdly, the email information does seem to save even if you get the `Required parameter missing: invitation` error. 

You can add this test to bottom of features/admins/admin_invitations.feature if you'd like: 
```
  Scenario: An admin can edit an invitation
    Given the user "dax" exists and is activated
      And "dax" has "2" invitations
      And I am logged in as an admin
    When I follow "Invite New Users"
      And I fill in "Enter a user name" with "dax"
      And I press "Go"
    Then I should see "copy and use"
    When I follow "Invite New Users"
      And I fill in "Enter an invite token" with "dax's" invite code
      And I press "Go"
    Then I should see "copy and use"
    When I fill in "invitation_invitee_email" with "oldman@ds9.com"
      And I press "Update Invitation"
    Then I should see "oldman@ds9.com" in the "invitation_invitee_email" input
```

When I run it, the failure doesn't look like the strong parameters errors we see on Travis, but it does fail on the last step, and the HTML screenshot it creates is a white page with a `Required parameter missing: invitation` error. (It does pass on master.)